fftest/vars.go

```diff
@@ -27,6 +27,42 @@ func Pair() (*flag.FlagSet, *Vars) {
 	return fs, &v
 }
 
+// EmptyValsFS returns a predefined flag set, and a predefined set of variables that
+// have been registered into it.
+// Custom Test Flagset creation function for the Empty (YAML) Values parsing, using more
+// specific flag names.
+func EmptyValsFS() (*flag.FlagSet, *Vars) {
+	fs := flag.NewFlagSet("ffemptytest", flag.ContinueOnError)
+
+	var v Vars
+	fs.StringVar(&v.S, "emptyStr", "", "string")
+	fs.IntVar(&v.I, "emptyInt", 0, "int")
+	fs.Float64Var(&v.F, "emptyFloat", 0.00, "float64")
+	fs.BoolVar(&v.B, "emptyBool", false, "bool")
+	fs.DurationVar(&v.D, "emptyDur", 0*time.Second, "time.Duration")
+	fs.Var(&v.X, "emptySlice", "collection of strings (repeatable)")
+
+	return fs, &v
+}
+
+func PresetValsFS() (*flag.FlagSet, *Vars) {
+	fs := flag.NewFlagSet("ffpresettest", flag.ContinueOnError)
+
+	//Set Defaults to be something OTHER than GoLang defaults for each type
```

Comment:
Why is this necessary?

Reply:
I wanted to ensure I was covering the "Negative-test scenario" that confirms that the No-Op for the Flagset update would not "overwrite" preset set of flags that are defined with something Other Than the Zero-Value for each type. 
  
So, this test fixture, `PresetValsFS`, in combination with test function `TestEmptyValsDontOverwritePresets()` ensure that test case is covered.

Reply:
`PresetValsFS() ` is no longer needed and has been removed.